docker-compose.dev.yml

```diff
@@ -4,9 +4,29 @@ services:
   db:
     build:
       context: .
-      dockerfile: Dockerfile_pg
+      dockerfile: Dockerfile_dev_pg
     volumes:
       - metadata2:/var/lib/postgresql/data
+    expose:
+      - "5432"
+    ports:
+      - "5432:5432"
+    networks:
+      - ddw_net
+
+  spotlights:
+    build:
+      context: .
+      dockerfile: Dockerfile_spotlights_dev
+    expose:
+      - "8000"
+    # Replace below path with the path to your ddw-external-api repo
+    volumes:
+      - "/Users/boss/chris/git/ddw-external-api:/src/ddw-external-api"
```

Comment:
I think this would work best here https://docs.docker.com/compose/environment-variables/